src/core/graph.js

```diff
@@ -589,6 +789,9 @@ export class Graph {
   }
 }
 
+/**
+ * Conert an edge into a human readable string.
```

Comment:
```
 * Conert an edge into a human readable string.
```

sp.: *Convert*

Need to specify that the precise behavior is an implementation detail
and is subject to change.

Should we specify that the function is injective?

Reply:
Fixed. I don't think we need to specify that it's injective.